Code/src/main/java/nl/utwente/group10/ui/components/blocks/output/ValueBlock.java

```diff
@@ -41,15 +41,21 @@
     public ValueBlock(CustomUIPane pane) {
         this(pane, "ValueBlock");
     }
+    
     protected ValueBlock(CustomUIPane pane, String fxml) {
         super(pane);
-
         value = new SimpleStringProperty("5.0");
-        output = new OutputAnchor(this, pane);
-
+        
         this.loadFXML(fxml);
+        try {
+            output = new OutputAnchor(this, new Value(new ConstT("Float"), getValue()).analyze(getPane().getEnvInstance()));
```

Comment:
Because at this point I have no idea how to use the backend, posponed for another PR.